static/app/views/organizationGroupDetails/allEventsTable.tsx

```diff
@@ -83,6 +81,7 @@ const AllEventsTable = (props: Props) => {
       excludedTags={excludedTags}
       projectId={projectId}
       totalEventCount={totalEventCount}
+      customColumns={['attachments', 'minidump']}
```

Comment:
Although not entierly used, this is in persuit of making the component more generic

Reply:
Or even better support the data from being fetched from discover in the first place!

Reply:
I'm wondering if it would be cleaner to handle everything inside the table component, so we keep the attachments and minidumps in the eventView object. This would abstract away the fact that these columns are custom columns, which would sound useful should we reuse this functionality elsewhere.